docs/projects/hello-world/_posts/2018-11-27-befunge.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Hello World in Befunge
+layout: default
+last-modified: 2018-11-27
+featured-image: hello-world-in-befunge-featured-image.JPEG
+tags: [befunge, hello-world]
+authors:
+  - stuin
+---
+
```

Comment:
Can you add the `# How to Implement the Solution` heading here along with an opening paragraph?